sites/all/modules/reol_review/reol_review.install

```diff
@@ -143,6 +143,31 @@ function _reol_review_schema_2() {
   return $schema;
 }
 
+/**
+ * Fourth schema version.
+ */
+function _reol_review_schema_3() {
```

Comment:
This scheme thing... is a hack. Also remeber to update `reol_review_schema()` so the real scheme match.

Reply:
Fixed.